test/controllers/event_controller_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Dwylbot.EventTestController do
+  use Dwylbot.ConnCase
+  alias Poison.Parser, as: PP
+
+  test "POST /event/new - no error", %{conn: conn} do
+    payload = "./test/fixtures/add_label.json"
+    |> File.read!()
+    |> PP.parse!()
```

Comment:
I've got a warning with I omit the brackets: ```warning: parentheses are required when piping into a function call.```
Maybe it's fine if there are not parameters?